src/main/java/build/buf/protovalidate/internal/celext/CustomOverload.java

```diff
@@ -305,6 +307,27 @@ private static Overload isUriRef() {
         });
   }
 
+  /**
+   * Creates a custom unary function overload for the "isNan" operation.
+   *
+   * @return The {@link Overload} instance for the "isNan" operation.
+   */
+  private static Overload isNan() {
+    return Overload.unary(
+        "isNan", value -> value.convertToNative(Double.TYPE).isNaN() ? BoolT.True : BoolT.False);
+  }
+
+  /**
+   * Creates a custom unary function overload for the "isInf" operation.
+   *
+   * @return The {@link Overload} instance for the "isInf" operation.
+   */
+  private static Overload isInf() {
+    return Overload.unary(
+        "isInf",
+        value -> value.convertToNative(Double.TYPE).isInfinite() ? BoolT.True : BoolT.False);
```

Comment:
don't want to be compatible with all the other impls? :-)

Reply:
resolved in [cd1cb84](https://github.com/bufbuild/protovalidate-java/pull/24/commits/cd1cb84568bf8a3179dcfa45f2aca39752458458)